[2025-02-10T23:49:54.379+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T23:49:54.419+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_generic_permissions_roles scheduled__2025-01-27T10:00:00+00:00 [queued]>
[2025-02-10T23:49:54.442+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_generic_permissions_roles scheduled__2025-01-27T10:00:00+00:00 [queued]>
[2025-02-10T23:49:54.443+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-10T23:49:54.476+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_generic_permissions_roles> on 2025-01-27 10:00:00+00:00
[2025-02-10T23:49:54.488+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=192) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T23:49:54.491+0000] {standard_task_runner.py:72} INFO - Started process 193 to run task
[2025-02-10T23:49:54.491+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_generic_permissions_roles', 'scheduled__2025-01-27T10:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp3vb26o_9']
[2025-02-10T23:49:54.494+0000] {standard_task_runner.py:105} INFO - Job 141: Subtask assign_generic_permissions_roles
[2025-02-10T23:49:54.618+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_generic_permissions_roles scheduled__2025-01-27T10:00:00+00:00 [running]> on host 9c5f21acce5a
[2025-02-10T23:49:54.759+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_generic_permissions_roles' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-27T10:00:00+00:00'
[2025-02-10T23:49:54.761+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T23:49:58.127+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:00.779+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:04.220+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:07.513+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:11.057+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:14.041+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:17.705+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:20.652+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:23.360+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:26.136+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:28.894+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:32.130+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:35.092+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:37.872+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:40.560+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:43.380+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:46.468+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:50.065+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:53.391+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:56.496+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:50:59.454+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:02.540+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:05.122+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-02-10T23:51:05.122+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-10T23:51:05.124+0000] {process_utils.py:132} INFO - Sending 15 to group 193. PIDs of all processes in the group: [827, 193]
[2025-02-10T23:51:05.124+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 193
[2025-02-10T23:51:05.124+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-02-10T23:51:05.132+0000] {dag.py:163} INFO - Error assigning DAG Runs permissions to data-team
[2025-02-10T23:51:05.137+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=827, status='terminated', started='23:51:01') (827) terminated with exit code None
[2025-02-10T23:51:08.098+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:11.073+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:15.086+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:19.132+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:23.058+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:27.134+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:31.676+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:36.444+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:40.843+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:45.291+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:49.678+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:55.217+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:51:59.169+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:52:03.172+0000] {dag.py:161} INFO - Assigning DAG Runs permissions to data-team
[2025-02-10T23:52:05.162+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=193, name='airflow task ru', status='sleeping', started='23:49:53') did not respond to SIGTERM. Trying SIGKILL
[2025-02-10T23:52:05.164+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 193
[2025-02-10T23:52:05.171+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=193, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='23:49:53') (193) terminated with exit code -9
[2025-02-10T23:52:05.172+0000] {standard_task_runner.py:190} ERROR - ('Job 141 was killed before it finished (likely due to running out of memory)', 'For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed')
